src/features/auth/signup/SetBirthday.tsx

```diff
@@ -244,6 +251,17 @@ export const SetBirthday: FunctionComponent<Props> = ({ route }) => {
   )
 }
 
+async function logBirthdayAnalytics(age: number) {
+  if (age >= 16) {
+    return
+  }
+  if (14 == age || age == 15) {
+    analytics.logSignUpBetween14And15Included()
+    return
+  }
+  analytics.logSignUpLessThanOrEqualTo13()
+}
```

Comment:
Proposition en moins de ligne et pour le même nombre de branchements:
```ts 
async function logBirthdayAnalytics(age: number) {
   if (age <= 13) {
     analytics.logSignUpLessThanOrEqualTo13()
   }
   else if (age === 14 || age === 15) {
     analytics.logSignUpBetween14And15Included()
   }
 }
```

question de curiosité: tu fais des Yoda-comparaison: `14 == age` ?

Reply:
à la base j'avais mis une inégalité puis je me suis ravisé^^
Ok pour moi sinon